content/relationships-or-mapping/reading/videos/_index.md

```diff
@@ -0,0 +1,21 @@
+---
+title: "Videos"
+date: 2022-12-15T09:16:07-06:00
+draft: false
+weight: 6
+originalAuthor: John Woolbright # to be set by page creator
+originalAuthorGitHub: jwoolbright23 # to be set by page creator
+reviewer: Kimberly Horan # to be set by the page reviewer
+reviewerGitHub: codinglikeagirl42 # to be set by the page reviewer
+lastEditor: # update any time edits are made after review
+lastEditorGitHub: # update any time edits are made after review
+lastMod: 12/15/22 # UPDATE ANY TIME CHANGES ARE MADE
+---
+
+The following video walkthroughs are all of the steps included in the one-to-many and many-to-many relationship sections.
+
+The one-to-many video uses the following github repository [orm-1 branch](https://github.com/LaunchCodeEducation/CodingEvents/tree/orm-1)
+{{< youtube 9mBJLk5eNnA >}}
```

Comment:
Sound on this video is too low, could barely hear with volume at maximum.  Also suggest moving this walkthrough to a separate page after creating a one-to-many relationship and before some-setup, the razor views chapter is a good example of video placement.